### ISSUE TITLE
"New tab" button moved randomly 

### ISSUE THREAD

**Issue by alexkuebo** [thedarkone/firepicker#20]
I installed Firepicker 1.4.1 with Firebug 1.9.1 on a Firefox 9.0.1 (Ubuntu 10.04.3 64bit) and now the button for opening new tabs is sometimes on the left and sometimes on the right side as if there are many open tabs. I will try to attach screenshots.
Btw: I tried it with a new profile and without other extensions

**Comment by alexkuebo** [comment]
Screenshots: http://imageshack.us/g/545/bildschirmfotomozillafi.png/

**Comment by thedarkone** [comment]
And if you remove Firepicker the weirdness is gone? Can you also try Firefox 10?

**Comment by alexkuebo** [comment]
Yes, without Firepicker the new tab button is always at its correct position. I'm sorry, Ubuntu 10.04 doesn't have a Firefox 10.

**Comment by SebastianZ** [comment]
I don't see that issue using FF 15.0.1 + FB 1.10.3 + FP 1.4.1 on WinXP.
Note that you can adjust the position of the new tab button while [customizing your toolbars](http://support.mozilla.org/en-US/kb/customize-firefox-controls-buttons-and-toolbars#w_how-do-i-customize-or-rearrange-toolbar-items). Does that fix your problem?

Sebastian
